### PR TITLE
DifficultWords Extraction Module

### PR DESCRIPTION
**Pull request by rpavani1998**
PR checklist:
- [x] Tested by creator locally
- [ ] Tested by creator on remote App
- [ ] Tested by reviewer locally
- [ ] Tested by reviewer on remote App
- [ ] Conforms with the agreed upon code pattern

### CODE REVIEW

**Review by LeonardPuettmannKern — CHANGES_REQUESTED** [review]
Looks great, thanks for creating the PR! I think the code snippet needs a few changes, I have already provided the code for that. :) Let me know if you have any questions.

File: extractors/python_functions/difficult_words_extraction/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        }
+
+
+def difficult_words_extraction(request: DifficultWordsExtractionModel):
+    """Extracts difficult words in a given text"""
+
+    text = request.text
+    syllable_threshold = request.syllable_threshold
+    difficult_words = textstat.difficult_words_list(text, syllable_threshold)
+    return {"difficult_words": difficult_words}
```
Comment (LeonardPuettmannKern) [inline extractors/python_functions/difficult_words_extraction/__init__.py:25]: We should also return the positon here, just as I explained it for the code_snippet.md

File: extractors/python_functions/difficult_words_extraction/code_snippet.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+YOUR_ATTRIBUTE: str =  "text"
+SYLLABLE_THRESHOLD: int = 2
+
+def difficult_words_extraction(record):
+    
+    text = record[YOUR_ATTRIBUTE].text
+    syllable_threshold = SYLLABLE_THRESHOLD
+    difficult_words = textstat.difficult_words_list(text, syllable_threshold)
+    
+    return difficult_words
```
[the remark this answers is not in the document]
Comment (rpavani1998) [reply extractors/python_functions/difficult_words_extraction/code_snippet.md:13]: Got it!

**Comment by rpavani1998** [comment]
Reopening the pr

**Pull request closed without merging**